dapp/src/utils/sentry.js

```diff
@@ -10,7 +13,7 @@ export const initSentry = () => {
   const config = getConfig()
   const distDir = `${config.serverRuntimeConfig.rootDir}/.next`
   Sentry.init({
-    enabled: process.env.NODE_ENV === 'production',
+    enabled: isProduction || isStaging,
     integrations: [
```

Comment:
I think we are missing the environment. See https://docs.sentry.io/platforms/javascript/configuration/environments/
Perhaps something like this:
```
      environment: isProduction ? 'production' : 'staging',
```

Reply:
Ah yes, my bad - added